pybikes/data/bysykkel.json

```diff
@@ -0,0 +1,34 @@
+{
+  "instances":[
+    {
+      "tag":"oslo-bysykkel",
+      "meta":{
+        "latitude":59.913869,
+        "country":"NO",
+        "name":"Oslo bysykkel",
+        "longitude":10.752245,
+        "city":"Oslo",
+        "city_code":"OSL"
+      },
+      "feed_url":"https://oslobysykkel.no/api/v1/stations",
+      "feed_details_url":"https://oslobysykkel.no/api/v1/stations/availability",
+      "feed_token":"b14e14eec39cccdf0140e728db2012b9"
```

Comment:
Such keys, unless this is somehow a public key and not a developer requested one, should not be packaged with the library.

Systems that require a key can be instanced with a key parameter (see velib example on the README). This key will be passed as `key` parameter on instantiation.